src/native/corehost/hostmisc/pal.h

```diff
@@ -269,6 +269,7 @@ namespace pal
 
     bool touch_file(const string_t& path);
     bool realpath(string_t* path, bool skip_error_logging = false);
+    bool fullpath(string_t* path, bool skip_error_logging = false);
```

Comment:
Please add a comment what is the difference between the two - the naming is Unix based, but Unix doesn't have `fullpath`, so it's unclear what that does.